guides/user/kubernetes-v2/automated-rollbacks/index.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: single
+title:  "Configure Automated Rollbacks in the Kubernetes Provider V2"
+sidebar:
+  nav: guides
+---
+
+{% include alpha version="1.6" %}
+
+{% include toc %}
+
+There are clear benefits to having your stored manifests (in Git, GCS, etc...)
+match as closely as possible to what's running in your cluster: you have
+auditability, versioned changes, and a high confidence in what code you have
+running. However, in some cases, pushing changes from someplace like GitHub
+through your build system all the way production can simply take too long, and
+an escape hatch is needed when something has gone seriously wrong.
```

Comment:
Do you mean something can go wrong _because of_ the time lag?

Can you give an example of something going wrong?

Reply:
I mean that once something has gone wrong, the time lag might be too long to be acceptable. There are always ways around this, but (for example) to build our clouddriver docker image, validate it, and (hypothetically) push it to production would take roughly 30 minutes. Simply clicking "rollback" takes less than 30 seconds.

Reply:
Ah, I'm getting it now. Maybe an additional sentence at the end: "Automated rollback is that escape hatch."